### PR TITLE
Follow-up to PR3764 deploying agent to the 'default' project

### PR DESCRIPTION
**Pull request by bmcelvee**
https://github.com/openshift/openshift-docs/pull/3764. @mwringe I opened this PR to catch the changes up with the documentation. Please take a look.

### CODE REVIEW

**Comment by mwringe** [comment]
LGTM

**Comment by bmcelvee** [comment]
@ahardin @adellape @mburke5678 peer review please 😺

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -570,6 +570,11 @@ information can be found under the
 link:https://github.com/hawkular/hawkular-openshift-agent[Hawkular OpenShift
 Agent documentation].
 
+The agent runs as a daemon set in your cluster and will be deployed to the
```
Comment (ahardin-rh) [inline install_config/cluster_metrics.adoc]: s/will be deployed/is deployed
s/will continue to be able to monitor/continues to monitor
s/plugin/plug-in

**Comment by ahardin-rh** [comment]
@bmcelvee Just a few minor nits from me. Otherwise, looks good! :star2:

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../install_config/cluster_metrics.adoc#install-config-cluster-metrics[Enabling Cluster Metrics]
|Added content on deploying the agent to the 'default' project in the xref:../install_config/cluster_metrics.adoc#deploying-hawkular-openshift-agent[Deploying the Hawkular OpenShift Agent] section.
%

**Pull request merged**